.github/pull_request_template.md

```diff
@@ -7,7 +7,5 @@
 Is this a breaking change? If yes, add notes below on why this is breaking and
 what additional work is required, if any.
 
-```
-[ ] Yes
-[ ] No
-```
+- [ ] Yes
+- [ ] No
```

Comment:
I don't really mind this since I don't think we have alternatives, but I've always thought that this utility wasn't great when it's supposed to be radio options not checkboxes (because then it always says 1 / 2 things complete). I vaguely remember there being a bigger discussion about this somewhere but I can't find it offhand.

Reply:
I've reverted them back to their original form. 👍🏽